polaris-service/src/main/java/org/apache/polaris/service/persistence/InMemoryPolarisMetaStoreManagerFactory.java

```diff
@@ -35,9 +35,10 @@
 import org.apache.polaris.core.persistence.PolarisTreeMapMetaStoreSessionImpl;
 import org.apache.polaris.core.persistence.PolarisTreeMapStore;
 
-@JsonTypeName("in-memory")
+@Named("in-memory")
+@Singleton
```

Comment:
We have to decide between `@Singleton` and `@ApplicationScoped` – unless we have a good reason to use the former, the latter is preferred.

Reply:
`@ApplicationScoped` is not part of the jakarta inject-api (it's in the enterprise package), so isn't supported by Dropwizard/HK2 today. I'm fine to remove this annotation and keep the scope defined in the configuration, but bringing in the jakarta enterprise-inject-api is a lot more work and I think we should keep this PR scoped

Reply:
Let's remove then, and let the CDI runtime decide what the scope is.